blank-canvas-blocks/sass/blocks/quote/_style.scss

```diff
@@ -0,0 +1,5 @@
+.wp-block-quote {
+	cite, .wp-block-quote__citation {
+		font-size: var(--wp--preset--font-size--small);
```

Comment:
Would you expect to be able to configure this?  Could this be a large font in another theme?  If so how would you expect to configure this once Gutenberg would allow for that?

There's no guarantee that a particular preset is going to be available in a child theme (since they can change those preset just like color). Maybe we should be leveraging only variables expressed in "custom".

Perhaps: `font-size: var(--wp--custom--quote--citation--font-size)`?

Reply:
> There's no guarantee that a particular preset is going to be available in a child theme (since they can change those preset just like color). Maybe we should be leveraging only variables expressed in "custom".

I see your point.

> Would you expect to be able to configure this? Could this be a large font in another theme? If so how would you expect to configure this once Gutenberg would allow for that?

As a theme developer and user, I would want to be able to change the font-size of the citation in a quote. I also expect the front-end and view to match. So to clarify for our purposes — should that be done with CSS + custom theme.json variables in this theme? 

And if so, the only difference with this approach compared to Seedlet is that I'm expressing the variables in the custom section of theme.json, rather than modular sass files.

Reply:
(Aside the II: That's the main difference yea.  There's some part in seedlet where the question to how to build the CSS was "how do we want it to work?" where the question here is rather "how do I expect to be able to configure that for Gutenberg and then take advantage of that configuration". I imagine the answer is often the same but likely not always.)

I think you're right. I expect that any CSS we add to this would be because it's something we're configuring in theme.json.  Hopefully little would be added that isn't directly linked to a variable.